src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -100,10 +138,11 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
       .times(marketData.price)
       .toFixed()
 
+    const apy = bnOrZero(
+      midgardPools.find(pool => pool.asset === thorchainPool.asset)?.saversAPR,
```

Comment:
Compiler safety for the sake of safety, since `Array.prototype.find()` on midgard pools data may return undefined, but we're actually bailing out of the flow earlier so it's safu